### DISCUSSION TITLE
Getting issue with edit mode/button of component

### DISCUSSION THREAD

**Discussion by Rahul139-stack** [websight-io/starter#129] [Q&A]
Getting issue with edit mode/button of individual component as after 1000ms edit button disappear.
Though it is in edit tab/mode while hovering the particular component ,edit button is not appearing .
**Sample**
![image](https://user-images.githubusercontent.com/81350384/216119462-3a60bd82-85a0-4c30-8693-3d0fa14bfddb.png)
**And Docker logs-**
![image](https://user-images.githubusercontent.com/81350384/216119801-0b5fd771-4236-4968-9f58-fc23d138742e.png)
So team, need some guide/ideas to overcome this issue. Thanks

**Comment by paweljozwicki** [comment]
Hi, so just to make sure. It's about the edit button that opens the dialog in which you can edit the component properties, right? 

If so, it happens for just one component or is it a more general problem?

**Reply by Rahul139-stack** [reply]
> Hi, so just to make sure. It's about the edit button that opens the dialog in which you can edit the component properties, right?
> 
> If so, it happens for just one component or is it a more general problem?
Yes @paweljozwicki  ,Edit button that opens the dialog to edit component properties, It happens for each and every components

**Reply by paweljozwicki** [reply]
Can you provide more details? What CMS version do you use? And you said that the edit button disappears after 1000ms. Does it mean that it's possible to edit the component earlier?

**Reply by Rahul139-stack** [reply]
Hi @paweljozwicki Actually our project is wrapped with docker file , when we run docker file in root directory of the project, edit mode is appearing perfectly, but when the docker file in the enviroment-local folder directory,edit mode disappear
